en/manage-prometheus-metrics.md

```diff
@@ -62,6 +73,18 @@ stanza like the following to the prometheus configuration:
 If the MAAS installation includes multiple nodes, the `targets` entries must be
 adjusted accordingly, to match services deployed on each node.
 
+If MAAS stats have also been enabled, an additional Prometheus job must added to the config:
+
+```yaml
+    - job_name: maas
+      metrics_path: /MAAS/metrics
+      static_configs:
+        - targets:
+          - <maas-host1-IP>:5240
+          - <maas-host2-IP>:5240
+          - <maas-host3-IP>:5240
```

Comment:
It doesn't matter how many hosts the MAAS deployment includes, you only need to add one of the IPs. Ideally you should add the load balancer host.

Having multiple entries like this will just make it harder to graph the metrics.